src/server/auth/server/testing/admin_test.go

```diff
@@ -516,6 +517,95 @@ func TestPreActivationPipelinesKeepRunningAfterActivation(t *testing.T) {
 	})
 }
 
+func TestPreActivationCronPipelinesKeepRunningAfterActivation(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	tu.DeleteAll(t)
+	defer tu.DeleteAll(t)
+	alice := robot(tu.UniqueString("alice"))
+	aliceClient, rootClient := tu.GetAuthenticatedPachClient(t, alice), tu.GetAuthenticatedPachClient(t, auth.RootUser)
+
+	// Deactivate auth
+	_, err := rootClient.Deactivate(rootClient.Ctx(), &auth.DeactivateRequest{})
+	require.NoError(t, err)
+
+	// Wait for auth to be deactivated
+	require.NoError(t, backoff.Retry(func() error {
+		_, err := aliceClient.WhoAmI(aliceClient.Ctx(), &auth.WhoAmIRequest{})
+		if err != nil && auth.IsErrNotActivated(err) {
+			return nil // WhoAmI should fail when auth is deactivated
+		}
+		return errors.New("auth is not yet deactivated")
+	}, backoff.NewTestingBackOff()))
+
+	// alice creates a pipeline
+	pipeline1 := tu.UniqueString("cron1-")
+	require.NoError(t, aliceClient.CreatePipeline(
+		pipeline1,
+		"",
+		[]string{"/bin/bash"},
+		[]string{"cp /pfs/time/* /pfs/out/"},
+		nil,
+		client.NewCronInput("time", "@every 10s"),
```

Comment:
could probably make this shorter to speed up the test, unless there's some effect of the delay I'm missing